teuthology/task/ceph_ansible.py

```diff
@@ -538,6 +542,83 @@ def run_rh_playbook(self):
         self.fix_keyring_permission()
         self.wait_for_ceph_health()
 
+    def run_haproxy(self):
+
+        """
+        task:
+            ceph-ansible:
+                haproxy: true
+                haproxy_repo: https://github.com/smanjara/ansible-haproxy.git
+                haproxy_branch: master
+        """
+        # Clone haproxy from https://github.com/smanjara/ansible-haproxy/,
+        # use inven.yml from ceph-ansible dir to read haproxy node from
+        # Assumes haproxy roles such as haproxy.0, haproxy.1 and so on.
+
+        installer_node = self.ceph_installer
+        haproxy_ansible_repo = self.config['haproxy_repo']
+        branch = 'master'
+        if self.config.get('haproxy_branch'):
+            branch = self.config.get('haproxy_branch')
+
+        installer_node.run(args=[
+            'cd',
+            run.Raw('~/'),
+            run.Raw(';'),
+            'git',
+            'clone',
+            run.Raw('-b %s' % branch),
+            run.Raw(haproxy_ansible_repo),
+        ],
+            timeout=4200,
+            stdout=StringIO()
+        )
+
+        remote = self.ctx.cluster.only(misc.is_type('haproxy', self.cluster_name)).remotes.iterkeys()
```

Comment:
this line can go away, there is a flake8 complaint for 'remote' not used.

Reply:
Addressed